src/post.proj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
```

Comment:
I'm fine with including it, I was just curious if there was a specific reason you needed it her. The default BuildAndTest targets is a good enough reason for me.